### ISSUE TITLE
Arrow keys not working on MacOS Sierra 10.12.5

### ISSUE THREAD

**Issue by aartiCEL** [strongloop/strongloop#344]
Hi
I'm trying to run slc loopback on my Mac Sierra machine, but it gets stuck on the version:

? What's the name of your application? 
? Which version of LoopBack would you like to use? (Use arrow keys)
  2.x (long term support) 
  3.x (current)

I am unable to use the arrow keys, or even move beyond this time. I could not even Ctrl-C it anymore beyond this, and finally have to kill the process from another terminal.

I tried installing loopback cli directly and strongloop/loopback-generator and use lb/ yo loopback but with the same results.

I have an older Mac running Yosemite, and I am not facing these issues on it... it seamlessly proceeds. Not sure how to proceed on my new machine.

Thanks so much.

**Comment by sam-github** [comment]
`slc` is superceeded by https://www.npmjs.com/package/loopback-cli, please use it instead

**Comment by aartiCEL** [comment]
Thanks so much! I had installed loopback-cli but that wasn't working either. 

I then downgraded my node to v.6.11.0 and installed apiconnect. Then somehow everything started working. Don't know whether it was because of downgrading the node version, or it had something to do with installing apiconnect -- as I did not check check in between executing the downgrade and installing apiconnect to see if it was working.
